html/index.html

```diff
@@ -395,52 +395,60 @@ <h2>About <br> Myself</h2>
         
         
             </ul>
+
+            <div class="section-background">
+
+                <div class="section">
         
-            <div class="section">
-        
-                <div class="contact-me-section">
-        
-                    <p id="contact-me">Contact me</p>
-        
-                    <p id="contact-msg"> 
-                        If you have an application you are 
-                        interested in developing, a feature 
-                        that you need built or a project that
-                        needs coding. I’d love to help with it 
-                    </p>
-        
-                </div>
-        
-        
-                <div class="form-section">
-        
-                    <form action="">
-        
-                        <input type="text" placeholder="Toyyib" class="text"> <br>
-                        <input type="email" placeholder="Simpleshaikh2@gmail.com" class="text"> <br>
-                        <textarea type="text" placeholder="write your message heres" id="type-msg"> </textarea><br>
-        
-                    </form>
-        
-                    <div class="btn-sec">
-        
-                        <button class="sec-btn">   
-                            Get in touch
-                        </button>
+                    <div class="contact-me-section">
+            
+                        <p id="contact-me">Contact me</p>
+            
+                        <p id="contact-msg"> 
+                            If you have an application you are 
+                            interested in developing, a feature 
+                            that you need built or a project that
+                            needs coding. I’d love to help with it 
+                        </p>
             
-                        <span class="dot"></span>
-        
                     </div>
+            
+            
+                    <div class="form-section">
+            
+                        <form action="https://formspree.io/f/mayzpyer" method="post">
+            
+                            <input type="text" name="name" placeholder="Name" class="text" maxlength="30" required> 
+                            <input type="email" placeholder="Email" class="text" id="text2" required> 
+                            <textarea name="textareaName" cols="30" rows="10" maxlength="500" id="type-msg" class="text" placeholder="Write your message here" required></textarea>
```

Comment:
- [optional] Great job making the changes. You can reduce the height of the textarea so it matches the figma design
![Screenshot (933)_LI](https://user-images.githubusercontent.com/65298048/229854534-e40e3c9b-2f01-47db-a6a9-b6c6d7fd2cb8.jpg)